pkg/modulewriter/tfwriter.go

```diff
@@ -64,9 +64,19 @@ func createBaseFile(path string) error {
 	return err
 }
 
+func handleBlueprintVariables(hclBytes []byte) []byte {
+	// Convert \$(not.variable) to $(not.variable)
+	re := regexp.MustCompile(`\\\\\$\((.*?)\)`)
+	return re.ReplaceAll(hclBytes, []byte(`$(${1})`))
+}
+
 func handleLiteralVariables(hclBytes []byte) []byte {
+	// Convert ((var.variable)) to var.variable
 	re := regexp.MustCompile(`"\(\((.*?)\)\)"`)
-	return re.ReplaceAll(hclBytes, []byte(`${1}`))
+	hclBytes = re.ReplaceAll(hclBytes, []byte(`${1}`))
+	// Convert \((not.variable)) to ((not.variable))
+	re = regexp.MustCompile(`\\\\\(\((.*?)\)\)`)
```

Comment:
Similar to above, you could call `escapeLiteralVariables` here since it performs the same task:
```go
...
hclBytes = re.ReplaceAll(hclBytes, []byte(`${1}`))
return escapeLiteralVariables(hclBytes)
```